### ISSUE TITLE
tf.keras.utils.Sequence generator

### ISSUE THREAD

**Issue by albertfaromatics** [zzh8829/yolov3-tf2#244]
Hi,

I'm trying to implement the training using a custom generator based on a Sequence.

I have everything ready but the part of the y_true/y_pred. I don't know I keep getting bad shapes on y_true when computing the loss.

How exactly the map on transform_targets work? Should I create a batch of images and annotations like:
x_train = [image1, image2, image3, ..., imageN]
y_train = [annotation1, annotation2, annotation3, ..., annotationN]

where
- each image is a (416, 416, 3) array
- each annotation is a (100, 5) array (with the padding funcion you have)

and then apply the "transform_targets" on the y_train before returning from __get_item__()??

Or when exactly the transform_targets should be applied?

I've been working on this for three days already and I'm going crazy.

Thanks

**Comment by albertfaromatics** [comment]
Any help here?
